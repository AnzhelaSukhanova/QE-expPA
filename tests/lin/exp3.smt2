(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult (bvshl #b0001 y) x) (= (bvshl #b0001 y) x)) 
 (or (bvult #b0010 x) (= #b0010 x)) 
 (or (bvult z x) (= z x)))))
