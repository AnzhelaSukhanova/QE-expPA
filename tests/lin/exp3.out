(let (($e1 (bvshl #b0001 y))) (let (($e2 (bvult #b0010 z))) (let (($e3 (bvult $e1 #b0010))) (let (($e4 (bvult z #b0010))) (let (($e5 (bvult $e1 z))) (let (($e6 (bvult #b0010 $e1))) (let (($e7 (bvult z $e1))) (not (and (not (and (not $e5) (not $e4))) (not (and (not $e3) (not $e2))) (not (and (not $e6) (not $e5))) (not (and (not $e7) (not $e2))) (not (and (not $e7) (not $e3))) (not (and (not $e6) (not $e4))))))))))))
